### DISCUSSION TITLE
实在用不了，申请退款

### DISCUSSION THREAD

**Discussion by garyguan1989** [getlantern/lantern-forum-cn#931] [General]
![微信图片_20241006112705](https://github.com/user-attachments/assets/69edc28b-05da-41ba-8fde-1ea21c03cc61)
实在用不了，申请退款，账号是349002735@qq.com

**Comment by garyguan1989** [comment]
没有人回复

**Reply by garyguan1989** [reply]
@Mike777Khoo

**Comment by garyguan1989** [comment]
349002735@qq.com

**Comment by garyguan1989** [comment]
@Mike777Khoo
请尽快协助退款

**Comment by MaxMKMax** [comment]
@garyguan1989 您好. 请问您愿意等我们一下吗? 我们会尽快修复好. 请您先别退款.

**Reply by garyguan1989** [reply]
请问什么时候可以退款，我不再使用

**Comment by garyguan1989** [comment]
什么时候退款

**Reply by MaxMKMax** [reply]
@garyguan1989 您好. 您的时间太久了. 应该退款不了了. 我们的购买协议是有些不允许退款的, 只是特殊情况我们才允许之前我有帮您提交问题上去了. 我是看帮您问看能否退款. 我尽力帮您询问看但不能保证.
